src/main/java/org/terasology/logic/health/HealthAuthoritySystem.java

```diff
@@ -140,16 +165,199 @@ private int calculateTotal(int base, TFloatList multipliers, TIntList modifiers)
 
     }
 
-    /** Handles DoHeal event to increase health of entity */
+    /**
+     * Handles DoHeal event to increase health of entity.
```

Comment:
Maybe tweak this with a code tag for `DoHealEvent` or even just avoid using the class name, instead referring to it generally "Event handler for increasing health ..."